docs/network/evm-nodes.md

```diff
@@ -58,26 +78,22 @@ The following instructions use the variable `<EVM_DATA_DIR>` to represent this d
      --dont-track-rollup-node
    ```
 
-   Alternatively, if you want to rely on a Smart Rollup node to check the correctness of blocks coming from the sequencer, pass the address of the Smart Rollup node to the `--rollup-node-endpoint` argument, as in this example:
+   If you set the `EVM_NODE_DATA_DIR` and `EVM_NODE_NETWORK` environment variables, you can omit the `--network` and `--data-dir` arguments, as in this example:
 
    ```bash
-   octez-evm-node init config \
-     --network testnet \
-     --data-dir <EVM_DATA_DIR> \
-     --rollup-node-endpoint <SR_NODE_OBSERVER_RPC>
+   octez-evm-node init config --dont-track-rollup-node
```

Comment:
This example does not use the environment variables unless they are implicit?

Maybe this would be clearer with
```sh
export EVM_NODE_DATA_DIR="<EVM_DATA_DIR>"
export EVM_NODE_NETWORK=mainnet
octez-evm-node init config --dont-track-rollup-node
```

or

```sh
EVM_NODE_DATA_DIR="<EVM_DATA_DIR>" EVM_NODE_NETWORK=mainnet octez-evm-node init config --dont-track-rollup-node
```

Reply:
Ok, added examples of using env variables.